packages/camera/lib/camera_image.dart

```diff
@@ -61,7 +61,15 @@ enum ImageFormatGroup {
   ///
   /// On iOS, this is `kCVPixelFormatType_32BGRA`. See
   /// https://developer.apple.com/documentation/corevideo/1563591-pixel_format_identifiers/kcvpixelformattype_32bgra?language=objc
+  ///
   bgra8888,
+
+  /// 32-big RGB image encoded into JPEG bytes.
+  ///
+  /// On Android, this is `android.graphics.ImageFormat.JPEG`. See
+  /// https://developer.android.com/reference/android/graphics/ImageFormat#FLEX_RGBA_8888
```

Comment:
You probably mean, `https://developer.android.com/reference/android/graphics/ImageFormat#JPEG`

Reply:
fixed on caf092e